libraries/chain/include/eosio/chain/fork_database.hpp

```diff
@@ -11,33 +11,33 @@ namespace eosio::chain {
    struct fork_database_impl;
 
    /**
-    * @class fork_database
+    * @class fork_database_t
     * @brief manages light-weight state for all potential unconfirmed forks
     *
     * As new blocks are received, they are pushed into the fork database. The fork
     * database tracks the longest chain and the last irreversible block number. All
     * blocks older than the last irreversible block are freed after emitting the
     * irreversible signal.
     *
-    * An internal mutex is used to provide thread-safety.
+    * Not thread safe, thread safety provided by fork_database below.
     */
    template<class bsp>  // either block_state_legacy_ptr or block_state_ptr
-   class fork_database {
+   class fork_database_t {
    public:
+      static constexpr uint32_t legacy_magic_number = 0x30510FDB;
+      static constexpr uint32_t magic_number = 0x4242FDB;
```

Comment:
Just curious. How were the magic numbers picked? Just random numbers?

Reply:
Yes, just random. I did stick with it ending in `FDB` which is cute.